index.html

```diff
@@ -1734,6 +1734,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: jafurman* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">jafurman</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/noTwitter" target="_blank">No Twitter</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/jafurman" target="_blank">Jafurman :3</a>
+          </p>
+          <p class="about">I like cats and code and games</p>
+          <div class="resources">
+            <p>0 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Resource 3</a>
+              </li>
```

Comment:
Please note that it is advisable to add any resources that could help the fellow developers! Kindly add some & commit back to get your PR merged.